azkaban-web-server/src/main/bash/azkaban-web-shutdown.sh

```diff
@@ -1,8 +1,15 @@
-azkaban_dir=$(dirname $0)/..
+#!/usr/bin/env bash
+# Shutdown script for azkaban web server
+installdir="$(dirname $0)/.."
 
-#!/bin/bash
-proc=`cat $azkaban_dir/currentpid`
-echo "killing AzkabanWebServer"
-kill $proc
+pid=`cat ${installdir}/currentpid`
+echo "Killing Web Server. [pid: $pid]"
 
-cat /dev/null > $azkaban_dir/currentpid
+kill ${pid}
+if [ $? -ne 0 ]; then
```

Comment:
I was noticing instances where shutdown wasn't shutting down the process in solo server.